src/getters/crunchbase.py

```diff
@@ -0,0 +1,114 @@
+"""
+discovery_utils.getters.crunchbase.py
+
+Module for easy access to downloaded CB data on S3.
+
+"""
+import pandas as pd
+import boto3
+import logging
+import re
+
+
+# Define global variables
+BUCKET_NAME = "discovery-iss"
+S3_PATH = "data/crunchbase/"
+AWS_ACCESS_KEY =
+AWS_SECRET_KEY = 
```

Comment:
these could be just `os.environ["AWS_ACCESS_KEY"]` and `os.environ["AWS_SECRET_KEY"]`

since aws cli should be configured, they would be fetched automatically anyways

Reply:
on S3_PATH, the context of crunchbase is given by the module name anyways. S3_PATH should be fine. But if this is referring to raw crunchbase, then the `raw` should be reflected here, as well as in the bucket paths as well.

Reply:
Ah, good point, thanks Solomon. I'll add `RAW` to these parts.